client/src/views/app.tsx

```diff
@@ -0,0 +1,10 @@
+import { Route, Routes } from "react-router-dom";
+import { Home } from "../views";
+
+export function App() {
+  return (
+    <Routes>
+      <Route index element={<Home />} />
```

Comment:
Just one route so far, with `/blocks` and `/transactions` and plenty more to come.